OpenTokSamples/7.Multiparty-ConstraintLayout/src/main/java/com/opentok/android/samples/multiparty_constraintlayout/ConstraintSetHelper.java

```diff
@@ -0,0 +1,79 @@
+package com.opentok.android.samples.multiparty_constraintlayout;
+
+import android.support.constraint.ConstraintLayout;
+import android.support.constraint.ConstraintSet;
+import android.transition.TransitionManager;
+
+/**
+ * Created by rpc on 11/04/2017.
+ */
+
```

Comment:
nit: Could you get rid of the header added by AS. We do not have them in any other sample so for the sake of having things the same way IMHO we should remote it.